packages/firestore/lite/src/api/field_value.ts

```diff
@@ -74,7 +74,7 @@ export function arrayUnion(...elements: unknown[]): FieldValue {
 }
 
 /**
- * Returns a special value that can be used with {@link setDoc} or {@link
+ * Returns a special value that can be used with {@link (setDoc:1)} or {@link
```

Comment:
@schmidt-sebastian you have to pick which overload to link to when linking to overloaded functions/methods

Reply:
Ack.